src/mds/MDCache.h

```diff
@@ -1114,7 +1114,7 @@ class MDCache {
   void handle_dentry_link(const cref_t<MDentryLink> &m);
   void handle_dentry_unlink(const cref_t<MDentryUnlink> &m);
 
-  int dump_cache(std::string_view fn, Formatter *f);
+  int dump_cache(std::string_view fn, Formatter *f, int64_t timeout);
```

Comment:
why not `double timeout`?

Reply:
I refer to the timeout types of some other functions...
changed it.
thx